internal/kubelet/client.go

```diff
@@ -11,23 +11,38 @@ import (
 	"net/http"
 	"net/url"
 	"os"
+	"path/filepath"
 	"strings"
 
 	"go.uber.org/zap"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/transport"
 
+	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/containerinsight"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/common/sanitize"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/k8sconfig"
 )
 
-const (
+var (
 	svcAcctCACertPath   = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
 	svcAcctTokenPath    = "/var/run/secrets/kubernetes.io/serviceaccount/token" // #nosec
 	defaultSecurePort   = "10250"
 	defaultReadOnlyPort = "10255"
 )
 
```

Comment:
is there anything we can do to avoid changing the `kubelet` directory?
We don't own this file so it is harder to upstream

If we change this we are going to have to get this upstreamed or deal with a fork which is not ideal

Reply:
After discussion in meeting, We settled on having this temporary workaround because it can be removed once AMIs have upgraded containerd to 1.7.  
Added comment in code to remove these temporary workarounds and cut ticket to EKS Windows team to work to removing these once upgraded AMIs are out.